code/modules/holodeck/computer.dm

```diff
@@ -396,6 +396,7 @@ GLOBAL_LIST_INIT(typecache_holodeck_linked_floorcheck_ok, typecacheof(list(/turf
 		return
 	for(var/obj/item/to_be_nerfed in spawned)
 		to_be_nerfed.damtype = nerf_this ? STAMINA : initial(to_be_nerfed.damtype)
+		to_be_nerfed.throwforce = nerf_this ? 0 : initial(to_be_nerfed.throwforce)
```

Comment:
again why

Reply:
Syndie decks are lethal when thrown, plus a lot of other objects can damage you which doesn't make any sense.

Reply:
does throwforce not respect damtype?

Reply:
I was wrong.  It does.  

https://github.com/tgstation/tgstation/blob/bb556b49a82c9bcded62202dccb0fe49efe0a574/code/modules/mob/living/living_defense.dm#L110-L120

I got confused after looking at dodgeballs where they were applying their stamina damage directly:

https://github.com/tgstation/tgstation/blob/321616b6816fbdadbc9cab7b77547446949272c0/code/modules/holodeck/items.dm#L69-L77

Reply:
Reverted this change with https://github.com/tgstation/tgstation/pull/64200/commits/4571fe4dee34f21ed4287ab19bbd8c9f841118d3